Dockerfile

```diff
@@ -1,26 +1,31 @@
-FROM rhel7
+# Using playbook2image as a base
+# See https://github.com/aweiteka/playbook2image for details on the image
+# including documentation for the settings/env vars referenced below
+FROM docker.io/aweiteka/playbook2image:latest
 
-MAINTAINER Troy Dawson <tdawson@redhat.com>
```

Comment:
Right, I applied the changes and force-pushed, but somehow github was still showing these outdated comments... the latest commit (`c6b43ef`) has dev@lists.openshift.redhat.com as maintainer.